boot/espressif/hal/CMakeLists.txt

```diff
@@ -29,9 +31,16 @@ list(APPEND INCLUDE_DIRS
     ${esp_idf_dir}/components/efuse/include
     ${esp_idf_dir}/components/efuse/${MCUBOOT_TARGET}/include
     )
+
+if("${MCUBOOT_ARCH}" STREQUAL "xtensa")
+    list(APPEND INCLUDE_DIRS        
```

Comment:
There is stray whitespace at the end of this line.

Reply:
Done

Reply:
I still see trailing whitespaces here.

Reply:
Sorry, fixed now.